tests/spec/cmdline_spec.rb

```diff
@@ -97,4 +97,16 @@ def self.pr_number
       expect(result).to be true
     end
   end
+
+  describe '.changed_since' do
+    it 'gitarro should see at least PR #30 changed in the last 60 seconds' do
+      context = 'changed-since'
+      desc = 'changed-since'
+      @pr = @rgit.pr_by_number(PR_NUMBER)
+      @comment = @rgit.create_comment(@pr, "gitarro rerun #{context} !!!")
```

Comment:
here i would write instead of @comment `comment`, since we don't need a class var

Reply:
Done, pending test and push.